### DISCUSSION TITLE
Sync Anki Review with Obsidian

### DISCUSSION THREAD

**Discussion by base-I** [ObsidianToAnki/Obsidian_to_Anki#164] [General]
Automatically Open the Note in obsidian, which your flashcard is related to. I think this would be an interesting piece of functionality - that way you could have obsidian running in the background - that way you could quickly refer to your notes in case you are uncertain about a concept or topic or want to see it in broader context (even (local) graph view) and how it is connected.

Loosing context has always posed to be the biggest problem with anki for me - this may be a remedy for that. Also I think it would fit into the scope of this plug in :)

**Comment by Pseudonium** [comment]
So there’s already a link back to the file added with Add File Link. Then, your request is to have the link opened automatically in Anki?

**Reply by base-I** [reply]
The idea is that obsidian automatically syncs with the note in obsidian - you could have them running side by side and obsidian opens the related note with each review, without any interaction by the user. That way obsidian would always be there on the correct note, whether it is needed or not.

It would interrupt the review in anki (= not focussing on obsidian). This does not mean that the link is automatically opened, rather a case of obsidian knowing which note is relevant and opening the respective note. I am not sure how this would be implemented but I think it would require anki to provide the relevant information as soon as the review note is opened.

**Reply by Pseudonium** [reply]
Right, I see. Since Anki supports JavaScript, I might be able to trigger obsidian opening the related note through a snippet. I’ll see what I can do.

**Reply by base-I** [reply]
I (as a "noob") had a look into the actions supported by anki connect; I think there is none to output the current note in review :( 
See here: https://github.com/FooSoft/anki-connect#supported-actions
